apps/gro/controllers/address.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const BaseController = require('hof').controllers.base;
+const ErrorController = require('hof').controllers.error;
+const _ = require('lodash');
+
+module.exports = class Address extends BaseController {
+  constructor(options) {
+    super(options);
+  }
+
+  getValues(req, res) {
+    res.locals.backLink = req.sessionModel.get('previous-step');
+    const addresses = req.sessionModel.get('addresses');
+    const formattedlist = _.map(_.map(addresses, 'formatted_address'), arg => arg.split('\n').join(', '));
+
+    const count = formattedlist.length + ' addresses';
+    this.options.fields.address.options = [count].concat(formattedlist);
+    super.getValues.apply(this, arguments);
```

Comment:
ES6 super methods automatically get called with the current scope so you don't need to use `call` or `apply` - this could be simplified to

```js
super.getValues(req, res)
```